### PR TITLE
add team and case ID for better visibility

### PR DESCRIPTION
**Pull request by yapei**
Add case ID and team name so that we can easily identify test results in ReportPortal

### CODE REVIEW

File: features/upgrade/web/console-upgrade.feature [hunk 1]
```diff
@@ -67,7 +67,7 @@ Feature: web console related upgrade check
   @network-ovnkubernetes @network-openshiftsdn
   @s390x @ppc64le @heterogeneous @arm64 @amd64
   @hypershift-hosted
-  Scenario: check console accessibility
+  Scenario: OCP-22597:UserInterface-check console accessibility
```
Comment (liangxia) [inline features/upgrade/web/console-upgrade.feature:70]: I think the `-` is unnecessary.

Comment (yapei) [reply features/upgrade/web/console-upgrade.feature:70]: thanks! good point

**Pull request merged**